.github/workflows/build_wheels.yml

```
@@ -46,7 +46,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-22.04, macos-12]
+        os: [ubuntu-22.04, macos-latest]
```

Not wrong, just curious: is there a reason to have MacOS at latest, but pin Ubuntu to a specific version?

Should this be
```suggestion
        os: [ubuntu-22.04, macos-15]
```
or
```suggestion
        os: [ubuntu-latest, macos-latest]
```
instead?

---

Also, another minor point, probably outside the scope of this ticket since I'm guessing this change is mainly about making an ARM build of SR for Mac rather than overhauling the CI, but iirc `ubuntu-24.04` is now the latest version for linux

---

Yeah, the idea was just to test that Mac ARM always works - it is difficult to stay up to date with all the versioning. Nevertheless, I agree that we should go for `ubuntu-24.04` and `macos-15`.